test/internal/evaluation/EvaluationInteractor.spec.ts

```diff
@@ -22,22 +22,21 @@ import { setupServerAndListen } from '../../utils'
 
 suite('internal/evaluation/EvaluationInteractor', () => {
   let server: SetupServer
+  let config: BKTConfig
   let component: DefaultComponent
   let interactor: EvaluationInteractor
   let evaluationStorage: EvaluationStorageImpl
 
   beforeEach(() => {
+    config = defineBKTConfig({
+      apiKey: 'api_key_value',
+      apiEndpoint: 'https://api.bucketeer.io',
```

Comment:
Could you rename this in another PR to `apiURL` to keep consistency with the other SDKs?

Reply:
You mean we are going to rename it everywhere?

https://github.com/bucketeer-io/android-client-sdk/blob/73508d1642da3c6db052f66a7471032aad56c269/bucketeer/src/main/kotlin/io/bucketeer/sdk/android/BKTConfig.kt#L17

Android Client SDK is currently `apiEndpoint`

Reply:
Ok, Swift SDK is `apiURL`

https://github.com/ca-dp/bucketeer-ios-sdk/blob/09e6a345135951fd2883bb7276d1abeac01f8efc/Bucketeer/Sources/Config.swift#L6


will do this in the next PR

Reply:
Sorry, it was correct. So you don't need to rename it.

Android 2.0
https://github.com/bucketeer-io/android-client-sdk/blob/main/bucketeer/src/main/kotlin/io/bucketeer/sdk/android/BKTConfig.kt#L17

iOS 2.0
https://github.com/ca-dp/bucketeer-ios-sdk/blob/v2/Bucketeer/Sources/Public/BKTConfig.swift#L5

Reply:
Sure, thanks for the quick update!